packages/suite-data/files/bin/coinjoin/linux-x64/WalletWasabi.WabiSabiClientLibrary.xml

```diff
@@ -0,0 +1,162 @@
+<?xml version="1.0"?>
+<doc>
+    <assembly>
+        <name>WalletWasabi.WabiSabiClientLibrary</name>
+    </assembly>
+    <members>
+        <member name="M:WalletWasabi.WabiSabiClientLibrary.Controllers.CryptographyController.SelectUtxoForRound(WalletWasabi.WabiSabiClientLibrary.Models.SelectUtxoForRoundRequest)">
+            <summary>
+            Given a set of unspent transaction outputs, choose a subset of the outputs that are best to register in a single CoinJoin round according to the given strategy.
+            </summary>
```

Comment:
Including this file is the only why I can make the binary work.
1. It is still not clear to me if we should include this files? 
2. Should it be included for all the binaries or only for `linux-x64`?
3. Should the binary be fix in a way that this file is not needed?

I created this issue https://github.com/trezor/coinjoin-backend/issues/11 in order to find the answers.